go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -260,9 +260,14 @@ func newSharedGatewayOpenFlowManager(patchPort, macAddress, gwBridge, gwIntf str
 				defaultOpenFlowCookie, ofportPatch, config.Default.ConntrackZone, ofportPhys))
 
 		// table0, Geneve packets coming from external. Skip conntrack and go directly to host
+		// if dest mac is the shared mac send directly to host.
+		dftFlows = append(dftFlows,
+			fmt.Sprintf("cookie=%s, priority=60, in_port=%s, dl_dst=%s, udp, udp_dst=%d"+
+				"actions=output:LOCAL", defaultOpenFlowCookie, ofportPhys, macAddress, config.Default.EncapPort))
```

Comment:
@adrianchiris  ideally there should be comma or space at the end of L265 just before the actions. I am surprised that it works when I tried this on my test node.

```
ovs-ofctl add-flow  brp0 'table=0 priority=10,udp,in_port=1,tp_dst=6088actions=output:3'
```
I think this would be more readable when grokking through the log
```
ovs-ofctl add-flow  brp0 'table=0 priority=10,udp,in_port=1,tp_dst=6088, actions=output:3'
```

Reply:
fully agree, will adjust.